lib/sync/datasetClientsCleaner.js

```diff
@@ -0,0 +1,78 @@
+var util = require('util');
+var parseDuration = require('parse-duration');
+var async = require('async');
+var _ = require('underscore');
+var DatasetClient = require('./DatasetClient');
+var syncUtil = require('./util');
+
+var storage;
```

Comment:
I think either way will work. The thing here is that I want `cleanDatasetClients` to be private. If I add it to the instance, it will become public.